src/usernotifications.cs

```diff
@@ -394,6 +451,23 @@ interface UNNotificationContent : NSCopying, NSMutableCopying, NSSecureCoding {
 		[Mac (10,15)]
 		[NullAllowed, Export ("targetContentIdentifier")]
 		string TargetContentIdentifier { get; [NotImplemented] set; }
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("contentByUpdatingWithProvider:error:")]
+		[return: NullAllowed]
+		UNNotificationContent ContentByUpdatingWithProvider (IUNNotificationContentProviding provider, [NullAllowed] out NSError outError);
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("interruptionLevel", ArgumentSemantic.Assign)]
+		UNNotificationInterruptionLevel InterruptionLevel { get; }
+
+		[NoWatch, NoTV, NoMac, NoiOS, MacCatalyst (15,0)]
+		[Export ("interruptionReason")]
+		string InterruptionReason { get; }
```

Comment:
that should be mapped to the catalyst-only enum you bound earlier

Reply:
Updated it! Then I saw B3 removed the enum type and these properties :(